src/status_im2/contexts/quo_preview/navigation/top_nav.cljs

```diff
@@ -2,48 +2,89 @@
   (:require [quo2.foundations.colors :as colors]
             [react-native.core :as rn]
             [reagent.core :as reagent]
-            [status-im2.common.home.view :as home.view]
-            [status-im2.contexts.quo-preview.preview :as preview]))
+            [quo2.core :as quo]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [status-im2.common.resources :as resources]
+            [quo2.theme :as quo.theme]))
 
 (def descriptor
-  [{:label   "Type"
-    :key     :type
+  [{:label   "Notification"
```

Comment:
I believe this could be shortened a bit in accordance with [quo preview refactorings](https://github.com/status-im/status-mobile/pull/16996)

Reply:
Sure I'll adjust this screen